src/psyclone/tests/domain/nemo/transformations/nemo_outerarrayrange2loop_trans_test.py

```diff
@@ -98,6 +98,65 @@ def test_transform_apply_mixed_implicit_do(tmpdir):
     assert Compile(tmpdir).string_compiles(result)
 
 
+def test_apply_with_structures(fortran_reader, fortran_writer):
+    '''Check that the PSyIR is transformed as expected when the
+    expressions contain a mix of arrays and structures in the same
+    reference.
+
+    '''
+    trans = NemoOuterArrayRange2LoopTrans()
+
+    # The inner dimension is already set
```

Comment:
As in the other test, I would have said this was 'outer' and the following one 'inner'.

Reply:
And you would have been right :) Fixed